src/www/ui/admin-dashboard.php

```diff
@@ -226,9 +226,12 @@ function DatabaseQueries()
     $head3 = _("Started");
     $head4 = _("Elapsed");
     $V .= "<tr><th>$head1</th><th>$head2</th><th>$head3</th><th>$head4</th></tr>\n";
-
-    $current_query = (strcmp($this->pgVersion['server'], "9.2") >= 0) ? "state" : "current_query";
-    $procpid = (strcmp($this->pgVersion['server'], "9.2") >= 0) ? "pid" : "procpid";
+    $getCurrentVersion = explode(" ", $this->pgVersion['server']);
+    $currentVersion = str_replace(".", "", $getCurrentVersion[0]);
+    $oldVersion = str_replace(".", "", "9.2");
+    unset($getCurrentVersion);
```

Comment:
maybe move some lines up, directly under 30?

Why is the `unset` necessary?

Reply:
because `$this->pgVersion['server']` creates string for postgres versions > 10
which looks like **10.0 (ubuntu 10.04 - ubuntu 18.04)** 
so when i explode this the array was creating 3 unnecessary values. apart from what i use. So used unset to empty the array once used.